docs/user/dashboard/make-dashboards-interactive.asciidoc

```diff
@@ -7,16 +7,16 @@
 :frontmatter-tags-content-type: [how-to] 
 :frontmatter-tags-user-goals: [analyze, visualize]
 
-Add interactive capabilities to your dashboard, such as interactive filter controls, and drilldowns that allow you to navigate to *Discover*, other dashboards, and external websites. 
+Add interactive capabilities to your dashboard such as drilldowns that allow you to navigate to *Discover*, other dashboards, and external websites. 
 
 ++++
 <script type="text/javascript" async 
 src="https://play.vidyard.com/embed/v4.js"></script>
 <img
 style="width: 100%; margin: auto; display: block;"
 class="vidyard-player-embed"
-src="https://play.vidyard.com/UhGkdJGC32HRn3oS5ZYJL1.jpg"
-data-uuid="UhGkdJGC32HRn3oS5ZYJL1"
+src="https://videos.elastic.co/watch/jqP5A4jNhbdtSyvBhrBgTw"
```

Comment:
The `src` property should point to a JPG. Unless there's a reason not to, let's use the vidyard URL structure. This is consistent with our other vids and will make it easier to search for the URL later.
```suggestion
src="https://play.vidyard.com/jqP5A4jNhbdtSyvBhrBgTw.jpg"
```

Reply:
Thank you for catching this and the explanation!